pkg/errors/types.go

```diff
@@ -42,4 +42,7 @@ var (
 
 	// ErrScanMetadata indicates problem with metadata in db.
 	ErrScanMetadata = New("failed to scan metadata in db")
+
+	//ErrSecretError indicates an error with the entity secret
+	ErrSecretError = New("invalid entity secret")
```

Comment:
With Gmail as an example

![image](https://github.com/mainflux/mainflux/assets/28790446/a190dae5-e14f-407c-8517-e51536075cbd)

Something like

```suggestion
	ErrWrongSecretError = New("wrong secret")
```

Reply:
Also, fix the comment. Add full stop at the end